gcc/rust/checks/errors/rust-unsafe-checker.h

```diff
@@ -115,7 +115,7 @@ class UnsafeChecker : public HIRFullVisitor
   virtual void visit (MatchExpr &expr) override;
   virtual void visit (AwaitExpr &expr) override;
   virtual void visit (AsyncBlockExpr &expr) override;
-  virtual void visit (InlineAsm &expr);
+  virtual void visit (InlineAsm &expr) override;
```

Comment:
I think that's part of #3244 so this needs a rebase I think?

Reply:
Deleted commit.